frontend/packages/pipelines-plugin/src/components/pac/PacAppForm.tsx

```diff
@@ -41,7 +41,9 @@ const PacAppForm: React.FC<FormikProps<FormikValues>> = ({
           <FormGroup
             label={t('pipelines-plugin~GitHub application name')}
             isRequired
-            helperText={t('pipelines-plugin~Provide a unique name')}
+            helperText={t(
+              'pipelines-plugin~Provide a unique name to your GitHub app e.g. "pipelines-ci-clustername" ',
```

Comment:
Is this space needed? :smirk: 

```suggestion
              'pipelines-plugin~Provide a unique name to your GitHub app e.g. "pipelines-ci-clustername" ,
```